app.js

```diff
@@ -89,18 +94,51 @@ webhooks.onAny((m) => {
             }
             if (m.name === "pull_request_review") {
                 if (m.payload.action === "submitted") {
-                    const user = m.payload.member;
                     const pr = m.payload.pull_request;
                     const review = m.payload.review;
+                    const user = m.payload.review.user;
 
                     const embed = new Discord.MessageEmbed();
                     embed.setColor('#0099ff');
-                    if (user) embed.setAuthor(user.login, user.avatar_url, user.html_url);
+                    embed.setAuthor(user.login, user.avatar_url, user.html_url);
                     embed.setURL(review.html_url);
-                    embed.setTitle(`New review for PR#${pr.number}`);
-                    if (review.body) {
-                        embed.setDescription(truncateString(review.body, pr));
+
+                    let title = `New review for PR #${pr.number} (${pr.title})`;
+                    if (review.state.toLowerCase() === "approved") {
+                        title = `PR #${pr.number} approved (${pr.title})`;
+                    } else if (review.state.toLowerCase() === "changes_requested") {
+                        title = `Changes requested to PR #${pr.number} (${pr.title})`;
                     }
+
+                    embed.setTitle(title);
+
+                    const comments = await octokit.request('GET /repos/{owner}/{repo}/pulls/{pull_number}/reviews/{review_id}/comments', {
```

Comment:
I think this s a little overkill for such a simple thing as counting the number of comments in the review. 

Do you really think it is worth it?

Reply:
It's not just for counting the comments; if the body of the review itself is empty and there is only one comment inside, then the body of that comment will be displayed. I think that does add something. In practice, that will be the case quite often, so it is convenient if you can read the comment directly in Discord instead of having to click a link.

Reply:
Gotcha let's try